### PR TITLE
feat(EMS-857): Account sign in - validation improvements

### PR DESCRIPTION
**Pull request by ttbarnes**
This PR improves the sign in account validation by rejecting a sign in attempt if the submitted credentials are valid, but the account is not verified. Also, display error messages in the form/page if the credentials are incorrect.

## Changes

- Update account sign in GQL resolver to reject if the account is not verified.
- Replace UI redirect to dummy `/invalid-credentials` URL with actual validation errors rendering in the page.
- Add E2E test coverage for all these scenarios.

### CODE REVIEW

File: e2e-tests/cypress/e2e/journeys/insurance/account/sign-in/validation/account-sign-in-validation.spec.js [hunk 1]
```diff
@@ -70,26 +70,25 @@ context('Insurance - Account - Sign in - Validation', () => {
     });
   });
 
-  // TODO: EMS-870 - create an account and enable these tests
-  // describe('when email does not match the created account, but password is correct', () => {
-  //   it('should render a validation error for both fields', () => {
-  //     cy.keyboardInput(accountFormFields[EMAIL].input(), `incorrect-${account[EMAIL]}`);
-  //     cy.keyboardInput(accountFormFields[PASSWORD].input(), account[PASSWORD]);
+  describe('when email and password is provided, but email is incorrect', () => {
+    it('should render a validation error for both fields', () => {
```
Comment (abhi-markan) [inline e2e-tests/cypress/e2e/journeys/insurance/account/sign-in/validation/account-sign-in-validation.spec.js:74]: Can we add a test when both `email` and `password` and incorrect please.

**Review by abhi-markan — CHANGES_REQUESTED** [review]
Good work, one minor comment to address please.

**Review by abhi-markan — APPROVED** [review]

**Review by Zainzzkk — DISMISSED** [review]
LGTM

**Review by Zainzzkk — APPROVED** [review]
LGTM

**Pull request merged**